test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -244,6 +244,65 @@
                             "found for service foo")}
               (json/parse-string (slurp (:body resp)))))))))
 
+
+(deftest simple-routes-params-ignoring-test
+  (with-status-service app
+    [foo-service]
+    (testing "ignores bad level"
+      (let [resp (http-client/get "http://localhost:8180/status/v1/simple?level=bar")]
+        (is (= 200 (:status resp)))
+        (is (= "running" (slurp (:body resp))))))
+    (testing "ignores alphabetic service_status_version"
+      (let [resp (http-client/get (str "http://localhost:8180/status/v1/"
+                                       "simple/foo?service_status_version=abc"))]
+        (is (= 200 (:status resp)))
+        (is (= "running" (slurp (:body resp))))))
+    (testing "ignores non-existent service_status_version"
+      (let [resp (http-client/get (str "http://localhost:8180/status/v1/"
+                                       "simple/foo?service_status_version=3"))]
+        (is (= 200 (:status resp)))
+        (is (= "running" (slurp (:body resp))))))))
+
+(deftest simple-routes-test
+  (testing "when calling the simple routes"
+    (testing "for all services"
+      (testing "and all services are :running"
+        (with-status-service app
+          [foo-service bar-service]
+          (let [resp (http-client/get "http://localhost:8180/status/v1/simple")]
+            (is (= 200 (:status resp)))
+            (is (= "running" (slurp (:body resp)))))))
+
+      (testing "and a service is :error"
+        (with-status-service app
+          [foo-service fail-service]
+          (let [resp (http-client/get "http://localhost:8180/status/v1/simple")]
+            (is (= 503 (:status resp)))
+            (is (= "error" (slurp (:body resp)))))))
+
+      (testing "and a service is :unknown"
+        (with-status-service app
+          [foo-service baz-service]
+          (let [resp (http-client/get "http://localhost:8180/status/v1/simple")]
+            (is (= 503 (:status resp)))
+            (is (= "unknown" (slurp (:body resp))))))))
```

Comment:
These tests seem a little sparse given that there's no distinction between the current behavior and what I think the behavior should be. Probably just a test of what happens when there's a :running service, an :error service, and a service that doesn't report its status correctly is enough to cover the gap.